### PR TITLE
Add floats and other numbers

### PR DESCRIPTION
**Pull request by Warry**

### CODE REVIEW

**Comment by Janiczek** [comment]
fixes #17

**Comment by Warry** [comment]
hum, I did not see that old commit sneaking in!

File: src/compiler/Stage/Emit/JavaScript.elm [hunk 1]
```diff
@@ -20,6 +20,9 @@ emitExpr ( expr, _ ) =
         Literal (Int int) ->
             String.fromInt int
 
+        Literal (Float float) ->
+            String.fromFloat float
```
Comment (Janiczek) [inline src/compiler/Stage/Emit/JavaScript.elm]: Let's add some emit tests for this. We can try for a little bit more corner cases than with [Ints](https://github.com/elm-in-elm/compiler/blob/master/tests/EmitTest.elm#L42-L46) - eg. what does parsing `0.3` become? etc.

Then we'll be able to :heavy_check_mark: the emit tests in the README table.

**Comment by Janiczek** [comment]
The diff in the PR is a bit weird now ... :thinking: maybe just some local cache. I'll try to deal with that.
